### DISCUSSION TITLE
起点中文互动投票页面右上角的关闭按钮变成了方块

### DISCUSSION THREAD

**Discussion by huangjiantang** [F9y4ng/GreasyFork-Scripts#284] [问答专区(Question / Answer)]
https://www.qidian.com/book/1037866670/

![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/25836998/54b43bc4-b876-4443-bd56-e63c1782ef71)

**Comment by F9y4ng** [comment]
在**需要渲染的网页元素**中，追加`:not(.global-icon em)`，保存为站点独享。
